commit 09dc2cdaa8e918fc89a2f80604b1b448f77a0b38 (refs/changes/43/1843/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-04-25T16:31:29-07:00 (2 years, 5 months ago)
    
    IMGAPI-623 `updates-imgadm -d ...` overly verbose: with node 0.10 it is serializing full "res" field

diff --git a/package.json b/package.json
index c3f3881..aeb43af 100644
--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
         "smartdc": "git+https://github.com/joyent/node-smartdc.git#4308fd9",
         "manta": "git+https://github.com/joyent/node-manta.git#b277acf",
         "moray": "git+https://github.com/joyent/node-moray.git#b84ef0e",
-        "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.2.3",
+        "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.2.5",
         "bunyan": "1.3.3",
         "assert-plus": "0.1.5",
         "ldapjs": "git+https://github.com/mcavage/node-ldapjs.git#f213b3e",
